README.md

```diff
@@ -1 +1,32 @@
-This is simple first project in microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# First Hello Microverse project
+
+>The first hello microverse project
+
+
+## Built With
+
+- HTML CSS
+- Linters
+
+
+## Getting Started
+
+**This is a public repository and could be used by anyone.**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Install
+ Node.js
+
+
+
+## Authors
+
+👤 **Suzana Marsela**
+
+- GitHub: [@githubhandle](https://github.com/Suzi216)
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add a [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) from the Microverse [README repo](https://github.com/microverseinc/readme-template). This will make sure this link goes to the LICENCE file because right now the link doesn't go anywhere